<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqClientMainWindow</class>
 <widget class="QMainWindow" name="pqClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsc/espina.qrc">
    <normaloff>:/espina/editor.ico</normaloff>:/espina/editor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="pqStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="statisticsDock">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Statistics Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqDataInformationWidget" name="statisticsView"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menuTools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QDockWidget" name="pipelineBrowserDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Pipeline Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPipelineBrowserWidget" name="pipelineBrowser">
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="volumeDock">
   <property name="windowTitle">
    <string>3D</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="volumeView"/>
  </widget>
  <widget class="QDockWidget" name="xzSliceDock">
   <property name="windowTitle">
    <string>X-Z</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="xzSliceView"/>
  </widget>
  <widget class="QDockWidget" name="yzSliceDock">
   <property name="windowTitle">
    <string>Y-Z</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="yzSliceView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openAction">
   <property name="text">
    <string>Open,,,,</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqPipelineBrowserWidget</class>
   <extends>QWidget</extends>
   <header>pqPipelineBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqDataInformationWidget</class>
   <extends>QWidget</extends>
   <header>pqDataInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqStatusBar</class>
   <extends>QStatusBar</extends>
   <header>pqStatusBar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rsc/espina.qrc"/>
 </resources>
 <connections/>
</ui>
